mslice/plotting/plot_window/plot_options.py

```diff
@@ -96,6 +117,8 @@ class SlicePlotOptions(PlotOptionsDialog):
 
     def __init__(self):
         super(SlicePlotOptions, self).__init__()
+        self._swap_widgets(self.gridLayout, self.chkXLog, self.chkXGrid, (2, 0), (2, 1))
+        self._swap_widgets(self.gridLayout_2, self.chkYLog, self.chkYGrid, (2, 0), (2, 1))
```

Comment:
I think we can avoid having to do this by putting the grid control on the left?

Reply:
Ok. That would be simpler!